Tests/UnitTests/Core/Geometry/CuboidVolumeBoundsTests.cpp

```diff
@@ -176,6 +176,32 @@ BOOST_AUTO_TEST_CASE(CuboidVolumeBoundarySurfaces) {
   BOOST_CHECK(pFaceZX.col(2).isApprox(yaxis));
 }
 
+BOOST_AUTO_TEST_CASE(CuboidVolumeBoundsSetValues) {
```

Comment:
This is the first set of the new style tests, I suppose.

Reply:
This just tests the setting of values. I'm planning to add construction tests when the functionality is actually added, like for the cylinder volume stack.